src/bundle/Controller/LocationController.php

```diff
@@ -395,4 +397,51 @@ public function addAction(Request $request): Response
             'locationId' => $contentInfo->mainLocationId,
         ]));
     }
+
+    /**
+     * Handles update existing location.
+     *
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function updateAction(Request $request): Response
+    {
+        $form = $this->formFactory->updateLocation(
+            new LocationUpdateData()
+        );
+        $form->handleRequest($request);
+
+        $location = $form->getData()->getLocation();
+
+        if ($form->isSubmitted()) {
+            $result = $this->submitHandler->handle($form, function (LocationUpdateData $data) {
+                $location = $data->getLocation();
+
+                $locationUpdateStruct = new LocationUpdateStruct(['sortField' => $data->getSortField(), 'sortOrder' => $data->getSortOrder()]);
+                $this->locationService->updateLocation($location, $locationUpdateStruct);
+
+                $this->notificationHandler->success(
+                    $this->translator->trans(
+                        /** @Desc("Location '%name%' updated.") */
+                        'location.update.success',
+                        ['%name%' => $location->getContentInfo()->name],
+                        'location'
+                    )
+                );
+
+                return new RedirectResponse($this->generateUrl('_ezpublishLocation', [
+                        'locationId' => $location->getContentInfo()->mainLocationId,
+                    ]) . '#ez-tab-location-view-details');
```

Comment:
I think you can use `_fragment` parameter instead of string concatenation: https://symfony.com/blog/new-in-symfony-3-2-routing-improvements

Reply:
Use magic parameter `_fragment` as @adamwojs said